zsh-aws-vault.plugin.zsh

```diff
@@ -6,6 +6,8 @@ AWS_VAULT_PL_CHAR=${AWS_VAULT_PL_CHAR:-$'\u2601'} # "the cloud"
 AWS_VAULT_PL_BROWSER=${AWS_VAULT_PL_BROWSER:-''}
 AWS_VAULT_PL_BROWSER_LAUNCH_OPTS=${AWS_VAULT_PL_BROWSER_LAUNCH_OPTS:-''}
 AWS_VAULT_PL_MFA=${AWS_VAULT_PL_MFA:-''}
+AWS_VAULT_PL_PERSIST_PROFILE=${AWS_VAULT_PL_PERSIST_PROFILE:-false}
+AWS_VAULT_PL_PERSIST_PROFILE_PATH=${AWS_VAULT_PL_PERSIST_PROFILE_PATH:-"$HOME/.config/zsh-aws-vault/avli-profiles"}
```

Comment:
#40 was creating this persistence in `~/.aws`. To avoid conflicting with the AWS CLI/SDK implementation, I prefer to keep these profiles in `~/.config/zsh-aws-vault`.